pkg/app/pipedv1/plugin/kubernetes/toolregistry/registry_test.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2024 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolregistry_test
```

Comment:
we usually not create test package, just keep it `package toolregistry`

Reply:
Fixed on https://github.com/pipe-cd/pipecd/pull/5246/commits/a6079c7bdac76905da0aadfd8c26564c4a31aab0